### DISCUSSION TITLE
Installation Problem with libxcrun

### DISCUSSION THREAD

**Discussion by LH3537** [Homebrew/discussions#3062] [Getting started]
Hi, I keep getting this error while trying to install.

`xcrun: error: unable to load libxcrun (dlopen(/Library/Developer/CommandLineTools/usr/lib/libxcrun.dylib, 0x0005): tried: '/Library/Developer/CommandLineTools/usr/lib/libxcrun.dylib' (mach-o file, but is an incompatible architecture (have 'x86_64', need 'arm64e')), '/usr/lib/libxcrun.dylib' (no such file)).
Failed during: git init -q`

Anyone know how to fix it?

**Comment by carlocab** [comment]
What is the output of `brew config` and `brew doctor`?

**Reply by LH3537** [reply]
command not found

**Reply by carlocab** [reply]
So you haven't installed Homebrew yet? Try doing
```
sudo rm -rf /Library/Developer/CommandLineTools
```
and then run the Homebrew installer again.

**Reply by TheBreznsoiza** [reply]
Oh thanks @carlocab this fixed my day and error 😊

**Reply by ricklesgibson** [reply]
Shout out to @carlocab: this also fixed my installer error, thank you

**Reply by themodrnhakr** [reply]
Finally! A solution to my problem!! Thanks my man!

**Reply by keithgardiner** [reply]
This also fixed an error I have had for a week! Thanks!
